### ISSUE TITLE
 bard.ai no longer working

### ISSUE THREAD

**Issue by tjl694** [hj91/node-red-contrib-bard#1]
hey, just to let you know, it looks like the bard.ai url is no longer working. navigating to that url results in a "buy this domain" kind of page. im not sure what the current url is though sadly

**Comment by hj91** [comment]
You are correct. This node needs to be recoded with proper url and api endpoints.
